charts/ndm/templates/03-deployment.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if .Values.ndmOperator.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "ndm.fullname" . }}-operator
```

Comment:
q: what is the difference between using template and include?. In configmap.yaml, include is used.